CONTRIBUTING.md

```diff
@@ -22,3 +22,12 @@ If you added a contributor by mistake, you can remove them in a comment with:
 
 If you are making a pull request on behalf of someone else but you had no part in designing the 
 feature, you can remove yourself with the above syntax.
+
+# Tests
+
+For normative changes, a corresponding
+[web-platform-tests](https://github.com/w3c/web-platform-tests) PR is highly appreciated. Typically,
```

Comment:
do you want to link directly to https://github.com/w3c/web-platform-tests/tree/master/gamepad?

Reply:
We're trying to use the same boilerplate text in all specs, but I can do that if you prefer.

Reply:
ah, makes sense - that's fine